# Post relative to covisible keyframes
FrameData frame
std_msgs/ByteMultiArray pose